camouflage/mocks/products/GET.mock

```diff
@@ -0,0 +1,162 @@
+
```

Comment:
looks like this is the only mock content that we really need so far.
please follow steps below:
- put move `camouflage/mocks/products/` into` __mocks__/products`
- in `__mocks__` folder create `config.yml` file (you can follow the original config but remove all the unnecessary rubbish in it)
- remove `camouflage` with the rest unnecessary rubbish in it. 

inside `__mocks__` folder you can also add notes about [how to install camouflage-server globally](https://www.npmjs.com/package/camouflage-server) and run mock server with our mocks in a `readme.md` file.

Reply:
done